cmd/view.go

```diff
@@ -179,6 +179,7 @@ func getTruncatedLine(line string, width int) string {
 	} else if lineLen < width {
 		return line + strings.Repeat(" ", width-lineLen)
 	} else {
-		return line[:width-3] + "..."
+		truncatedRunes := []rune(line)[:width-3]
```

Comment:
This is great. Thank you!

Reply:
You're very welcome. I love GoCSV, it's made my work so much easier!